_pages/clients/scalastr.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Scalastr
+permalink: /scalastr/
+web: 
+github: benthecarman/scalastr
+instances:
+tags:
+- social
+- client
+platforms:
+- cross-platform
```

Comment:
list those platforms, please. If it works on Android, "android" should be listed so filtering works. People want to know if it works on their Android.

Reply:
For many of the projects, it's hard to determine the exact platforms. Especially if it doesn't have a readme file. What do I do in cases where I can't identify exactly what platform it supports?

Reply:
Messaged benthecarman. I hope I don't annoy him, dude has 98 repositories!